lightwood/mixer/regression.py

```diff
@@ -12,6 +12,8 @@
 
 
 class Regression(BaseMixer):
+    '''
```

Comment:
might conflict with george's docs

Reply:
these are my docs too :p will fix conflict if it arises, not sure why this is here.